tests/test_copyoffload_migration.py

```
@@ -706,3 +706,175 @@ def test_copyoffload_rdm_virtual_disk_migration(
 
     # Verify that the correct number of disks were migrated (1 base + 1 RDM = 2)
     verify_vm_disk_count(destination_provider=destination_provider, plan=plan, target_namespace=target_namespace)
+
+
+@pytest.mark.copyoffload
+@pytest.mark.parametrize(
+    "plan,multus_network_name",
```

Ruff (0.14.10)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)